### PR TITLE
Update pricing.mdx + add container_types.mdx

### PR DESCRIPTION
**Pull request by Michellehbn**
added note on upcoming unavailability of c6i instances

And added new page to list container types

updated FAQ

### CODE REVIEW

File: docs/source/container_types.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Inference Endpoints Container Types
+
+When you create an Endpoint, you have a variety of options when selecting a container type.
+
+## Default
+
+The default container type is the easiest way to deploy endpoints and is flexible thanks to [custom Inference Handlers](https://huggingface.co/docs/inference-endpoints/guides/custom_handler).
```
Comment (philschmid) [inline docs/source/container_types.mdx:7]: Should we link to the toolkit as it is now public?

**Review by philschmid — APPROVED** [review]

File: docs/source/container_types.mdx
Comment (philschmid) [inline docs/source/container_types.mdx:1]: also if you want a new page to show up you need it to add it to https://github.com/huggingface/hf-endpoints-documentation/blob/main/docs/source/_toctree.yml

File: docs/source/container_types.mdx (hunk 1 above)
Comment (Michellehbn) [reply docs/source/container_types.mdx]: added!

File: docs/source/container_types.mdx
Comment (Michellehbn) [reply docs/source/container_types.mdx:1]: added!

**Review by philschmid — APPROVED** [review]

**Review by pagezyhf — APPROVED** [review]

**Pull request merged**